lib/Driver/Driver.cpp

```diff
@@ -700,7 +700,9 @@ computeWorkingDirectory(const llvm::opt::InputArgList *ArgList) {
     SmallString<128> workingDirectory;
     workingDirectory = A->getValue();
     llvm::sys::fs::make_absolute(workingDirectory);
-    return workingDirectory.str();
+    //  Create a new string that can outlive ArgList.
```

Comment:
It's not about outliving ArgList; it's outliving the stack-based `workingDirectory`.

Reply:
Sigh.. yes! Fixed!!. Thanks.